stdlib/public/core/Array.swift

```diff
@@ -1800,6 +1801,8 @@ extension Array {
   ///     }
   ///     // numbers == [1, 2]
   ///
+  /// [1]: https://developer.mozilla.org/en-US/docs/Glossary/Endianness
```

Comment:
This would be the first link to Mozilla developer documentation in the repository.  Elsewhere, we have links to Wikipedia.  Would this article work instead?

https://en.wikipedia.org/wiki/Endianness

Reply:
I asked one of the editors about this link in the context of the writing style we use in documentation. To follow that style, like the rest of the standard library documentation aims to do, we should omit the link.

Reply:
I am not sure I understand. Should link be removed???

Reply:
Yes.

Reply:
Done